conf/messages

```diff
@@ -479,7 +479,8 @@ calc.resident.disposal.value.question = How much did you sell the property for?
 
 ## Property Worth when you sold it messages
 calc.resident.properties.worthWhenSoldForLess.question = What was the property worth when you sold it?
-calc.resident.properties.worthWhenSoldForLess.extraText = You can use a valuation from a surveyor or a property website.
+calc.resident.properties.worthWhenSoldForLess.hintText = If you owned the property with someone else, only enter your share of the property value.
```

Comment:
The hint text should be "If you owned the property with someone else, only enter the value of your share of the property."